tests/test_main.py

```
@@ -0,0 +1,26 @@
+
+
+def test_module_execution():
+    """Ensure `python -m vector_topic_modeling` executes the CLI."""
+    with (
+        patch.object(sys, "argv", ["vector_topic_modeling", "cluster", "--help"]),
+        patch("vector_topic_modeling.cli.main", return_value=0) as mock_main,
+    ):
+        try:
+            runpy.run_module("vector_topic_modeling", run_name="__main__")
+        except SystemExit as e:
+            assert e.code == 0
```

Ruff (0.15.7)

[warning] Found assertion on exception `e` in `except` block, use `pytest.raises()` instead

(PT017)